### PR TITLE
Add `stylish` as `ansi_term` alternative

### PR DESCRIPTION
**Pull request by Nemo157**

### CODE REVIEW

File: crates/ansi_term/RUSTSEC-2021-0139.md [hunk 1]
```diff
@@ -26,7 +26,8 @@ Last release seems to have been three years ago.
  - [nu-ansi-term](https://crates.io/crates/nu-ansi-term)
  - [owo-colors](https://crates.io/crates/owo-colors)
  - [yansi](https://crates.io/crates/yansi)
+ - [stylish](https://crates.io/crates/stylish)
```
Comment (pinkforest) [inline crates/ansi_term/RUSTSEC-2021-0139.md:29]: We use alphabetical sorting, could we please move it before yansi and after owo-colors thx

File: crates/ansi_term/RUSTSEC-2021-0139.md [hunk 2]
```diff
@@ -26,7 +26,8 @@ Last release seems to have been three years ago.
...
 
 ## Dependency Specific Migration(s)
 
- - [structopt, clap2](https://github.com/clap-rs/clap/discussions/4172)
\ No newline at end of file
+ - [structopt, clap2](https://github.com/clap-rs/clap/discussions/4172)
```
Comment (pinkforest) [inline crates/ansi_term/RUSTSEC-2021-0139.md:33]: I wonder what happened here.. did it add some extra spaces at end or smth ?

**Comment by pinkforest** [comment]
Thanks for the contrib @Nemo157 just couple of nits pls :) thx

File: crates/ansi_term/RUSTSEC-2021-0139.md (hunk 2 above)
Comment (Nemo157) [reply crates/ansi_term/RUSTSEC-2021-0139.md]: It's adding a trailing newline (for some reason github doesn't render the `No newline at end of file` marker that most diff tools do). I should probably set `nofixendofline` for contributing to repositories that don't force it.

Comment (pinkforest) [reply crates/ansi_term/RUSTSEC-2021-0139.md]: Cool wondered why I didn't see the marker, thx

**Pull request merged**